__tests__/html/typing/layout.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en-US">
   <head>
     <link href="/assets/index.css" rel="stylesheet" type="text/css" />
-    <script crossorigin="anonymous" src="https://unpkg.com/@babel/standalone/babel.min.js"></script>
+    <script crossorigin="anonymous" src="https://unpkg.com/@babel/standalone@7.26.5/babel.min.js"></script>
```

Comment:
7.8.7?

Reply:
While most are ok at 7.8.7, others do fail, so I had to bump some to the different version